src/gsi/setupoz.f90

```diff
@@ -1603,13 +1626,6 @@ subroutine init_netcdf_diag_
         call nc_diag_header("Satellite_Sensor", isis)
         call nc_diag_header("Satellite", dplat(is))
         call nc_diag_header("Observation_type", obstype)
-        call nc_diag_header("pobs", pobs)
```

Comment:
lines 1606-1612 need to be reverted.

Reply:
@HaixiaLiu-NOAA Many thanks for more changes to make. I'll get these done in the next few days. @jswhit It looks like the lines being added from the NCEP version of setupoz were probably never in this NASA version to begin with. The two must've developed separately for a while.

Reply:
@jswhit Haixia's changes to setupoz are now in hera:/scratch1/NCEPDEV/global/Jack.Woollen/rrgsi.